rs/tests/driver/src/driver/ic.rs

```diff
@@ -291,6 +313,15 @@ impl InternetComputer {
                     .ipv6,
             );
         }
+        for n in self.api_boundary_nodes.iter_mut() {
```

Comment:
And again, here you could use the same simplification has [above](https://github.com/dfinity/ic/pull/2290#discussion_r1820939276) and just have a single for-loop iterating over all nodes.

Reply:
Thanks for pointing it out! I updated it.